.buildkite/pipeline.yml

```diff
@@ -11,6 +11,12 @@ steps:
             - ruby-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/ruby:base-ruby${BRANCH_NAME}
             - ruby-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/ruby:base-ruby-latest
 
+  - name: ':copyright: License Audit'
+    plugins:
+      docker-compose#v3.7.0:
+        run: license_finder
+    command: /bin/bash -lc '/scan/scripts/license_finder.sh'
```

Comment:
Can we either put this in the first or second group of test steps rather than here? Otherwise we won't get feedback from CI if we introduce a dependency that doesn't pass the audit, until it's approved

Reply:
Yes, that makes sense.